src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -77,14 +83,14 @@ private void createHearing(HearingWrapper wrapper) {
 
 
     private void updateHearing(HearingWrapper wrapper) {
-        // TODO implement mapping for the event when the hearing's details are updated
+        sendUpdateHearingRequest(wrapper);
```

Comment:
updateIds(wrapper);
buildRelatedParties(wrapper);
Needs to be added before sendUpdateHearingRequest, as the parties many have changed.